tests/integration/test_base_model.py

```diff
@@ -52,7 +52,7 @@ def test_no_update(self):
 
         assert obj.first_name == 'Bye'
         assert obj.last_name == None
-        assert obj.updated_at == None
+        assert obj.updated_at != None
```

Comment:
This test can be deleted. It's not testing any behavior that is not already under test, and failed because it was testing an implementation setup.

Reply:
Also if would be awesome if you could just delete `assert obj.updated_at != None` on lines 39 and 47 for me as well.

Reply:
Thanks for finding these!